wiki/Tournaments/Donut_Cup/1/en.md

```diff
@@ -350,6 +380,8 @@ Sunday, 7 July 2024:
 
 ### Semifinals
 
+Detailed statistics for this round in [the following spreadsheet](https://docs.google.com/spreadsheets/d/1lmZEZ5WxtXXi3F-cHqmo5wmc0LbbsJVKl7RY0y9HbRs/edit).
```

Comment:
is this the wording used elsewhere on the wiki? "the following spreadsheet" sounds to me like there should be a spreadsheet below this sentence

Reply:
changed to "this spreadsheet"